irohad/main/application.cpp

```diff
@@ -31,7 +31,9 @@
 #include "torii/impl/status_bus_impl.hpp"
 #include "validators/default_validator.hpp"
 #include "validators/field_validator.hpp"
+#include "validators/proposal_validator.hpp"
```

Comment:
Is there any sense to import both interface and proto parts? Just proto should be enough